docs/source/intro_to_nwb/3_basic_neurodata_types.rst

```diff
@@ -0,0 +1,81 @@
+properties of `TimeSeries` but additionally specifies that the "data" dataset must be 2-dimensional, where the second
+dimension is electrode, and it must contain an additional "electrodes" dataset with row indices into the "electrodes"
+table of the NWB file to indicate which electrodes correspond to the second dimension of the "data" dataset.
+
+.. seealso::
+    For your reference, NWB defines the following main :ref:`nwb-schema:sec-TimeSeries` subtypes:
+
+    * **Extracellular electrophysiology:**
+      :ref:`nwb-schema:sec-ElectricalSeries`, :ref:`nwb-schema:sec-SpikeEventSeries`
+    * **Intracellular electrophysiology:**
+      :ref:`nwb-schema:sec-PatchClampSeries` is the base type for all intracellular time series, which
+      is further refined into subtypes depending on the type of recording:
+      :ref:`nwb-schema:sec-CurrentClampSeries`,
+      :ref:`nwb-schema:sec-IZeroClampSeries`,
+      :ref:`nwb-schema:sec-CurrentClampStimulusSeries`,
+      :ref:`nwb-schema:sec-VoltageClampSeries`,
+      :ref:`nwb-schema:sec-VoltageClampStimulusSeries`.
+    * **Optical physiology and imaging:** :ref:`nwb-schema:sec-ImageSeries` is the base type
+      for image recordings and is further refined by the
+      :ref:`nwb-schema:sec-ImageMaskSeries`,
+      :ref:`nwb-schema:sec-OpticalSeries`, and
+      :ref:`nwb-schema:sec-TwoPhotonSeries` types.
+      Other related time series types are:
+      :ref:`nwb-schema:sec-IndexSeries` and
+      :ref:`nwb-schema:sec-RoiResponseSeries`.
+    * **Others:** :ref:`nwb-schema:sec-OptogeneticSeries`,
+      :ref:`nwb-schema:sec-SpatialSeries`,
+      :ref:`nwb-schema:sec-DecompositionSeries`,
+      :ref:`nwb-schema:sec-AnnotationSeries`,
+      :ref:`nwb-schema:sec-AbstractFeatureSeries`, and
+      :ref:`nwb-schema:sec-IntervalSeries`.
+
+All time values in a `TimeSeries` and other neurodata types must be stored in seconds relative to the
+``timestamps_reference_time`` value, a datetime value stored at the root of the NWB file.
+By default, this is the same as the ``session_start_time``,
+another datetime value stored at the root of the NWB file.
+
+DynamicTable
+-------------
+Tabular (table-like) data are stored in :ref:`hdmf-common-schema:sec-DynamicTable` objects,
+which are column-based tables to which you can add custom columns.
+
+Similar to `TimeSeries`, several neurodata types inherit from `DynamicTable` that are specialized
+for particular types of data and specify particular required or optional columns.
+
+.. seealso::
+    For your reference, NWB defines the following main :ref:`hdmf-common-schema:sec-DynamicTable` subtypes:
+
+    * :ref:`nwb-schema:sec-TimeIntervals`: stores trials, epochs, and associated metadata.
+    * :ref:`nwb-schema:sec-Units`: stores spike times of sorted units and associated metadata.
+    * :ref:`nwb-schema:sec-PlaneSegmentation`: stores regions of interest for optical imaging with associated
+      metadata.
```

Comment:
I think we should also add:
* One Item for the Intracellular Ephys tables: i.e., IntracellularRecordingsTable, SimultaneousRecordingsTable, SequentialRecordingsTable, RepetitionsTable, ExperimentalConditionsTable. 
* https://nwb-schema.readthedocs.io/en/latest/format.html#intracellularrecordingstable

* https://nwb-schema.readthedocs.io/en/latest/format.html#groups-general-extracellular-ephys-electrodes